src/adv.py

```diff
@@ -47,5 +48,59 @@
 #
 # If the user enters a cardinal direction, attempt to move to the room there.
 # Print an error message if the movement isn't allowed.
-#
+
 # If the user enters "q", quit the game.
+name = input("Tell me your name, Human: ")
+
+#current room name
+player = Player(name, room["outside"])
+print(f"Hello {name}. Walk into the rooms and collect the coins. You are currently in the {player.room.name}")
+
+while True:
+    directions = input("Please pick a direction using:  'n' , 's' , 'w', 'e', 'q to quit'")
+    print(player.room.description)
+
+    movement = input("Choose year direction: ")
+     
+    walking = directions[movement]
+    
+    if directions == "n" or directions == "N":
```

Comment:
This was a creative way to check for case, another way would be to have the input be set to either `upper` or `lower`, then you would be checking case in one place vs repeating it multiple times